tshirts.cpp

```diff
@@ -14,6 +14,7 @@ char size(int cms) {
 }
 
 int main() {
+    assert(size(0) == 'S');

```

Comment:
Does this assert convey a user's need? What would a t-shirt size of `0` mean? Perhaps it means that they entered something by mistake... is it good to ship a 'small' size t-shirt to them?